torch/testing/_internal/common_methods_invocations.py

```diff
@@ -18675,6 +18681,24 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
             ),
         ),
     ),
+    OpInfo(
+        'hann_window',
+        aten_name='hann_window',
+        dtypes=_dispatch_dtypes((
+            torch.int32, torch.int8, torch.float16, torch.complex64, torch.bool,
+            torch.complex128, torch.int64, torch.bfloat16, torch.float64,
+            torch.int16, torch.float32, torch.uint8)),
+        dtypesIfCUDA=_dispatch_dtypes((
+            torch.int32, torch.int8, torch.float16, torch.complex64, torch.bool,
+            torch.complex128, torch.int64, torch.bfloat16, torch.float64,
+            torch.int16, torch.float32, torch.uint8)),
+        dtypesIfROCM=_dispatch_dtypes((
+            torch.int32, torch.int8, torch.float16, torch.complex64, torch.bool,
+            torch.complex128, torch.int64, torch.bfloat16, torch.float64,
+            torch.int16, torch.float32, torch.uint8)),
```

Comment:
```suggestion
        dtypes=all_types_and(torch.bool, torch.half, torch.bfloat16),
```

Reply:
Why do we want to remove aten_name?

Reply:
`aten_name` defaults to the same as `name` so there's not need to specify it here.

Reply:
we just need aten_name if it's not the same as that of the opinfo